.github/workflows/rubyonrails.yml

```
@@ -70,28 +88,34 @@ jobs:
+
       - name: Static type checking
-        run:  |
+        run: |
           bundle exec rbs collection install
           bundle exec steep check
+
   lint:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)